### ISSUE TITLE
Web component: Card group - when focus reaches the link in each card, Screen reader announces a lengthy text which makes it over verbose.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6512]
## Detailed description

Issue:
In tab / shift tab navigation, when focus reaches the link in each card, Screen reader announces a lengthy text which makes it over verbose.

Expected:
It may be possible if the entire content in the card is given in anchor tag. In this scenario, only the heading can be given in anchor tag. Note: the link text/label should be concise and convey only the relevant information.

**Comment by Praveen-sr** [comment]
Ram re tested this on both JAWS and Voiceover on the [ link ]( https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/6642/index.html?path=/story/overview-getting-started--page)  - Working as expected.
